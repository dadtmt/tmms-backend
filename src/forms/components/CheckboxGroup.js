import React, { PropTypes } from 'react'
import { Checkbox } from 'react-bootstrap'

const CheckboxGroup = ({ input, label }) => (
    <Checkbox {...input} checked={input.value} >{label}</Checkbox >
)

CheckboxGroup.propTypes = {
  input: PropTypes.object.isRequired,
  label: PropTypes.string.isRequired
}

export default CheckboxGroup
