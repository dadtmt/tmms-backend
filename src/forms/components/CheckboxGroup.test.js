import React from 'react'
import renderer from 'react-test-renderer'

import CheckboxGroup from './CheckboxGroup'

it('should render correctly', () => {
  const props = {
    input: {
      value: false
    },
    label: 'Check this'
  }
  expect(
    renderer.create(<CheckboxGroup {...props} />).toJSON()
  ).toMatchSnapshot()
})
