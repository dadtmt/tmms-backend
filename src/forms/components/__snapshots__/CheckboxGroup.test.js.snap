exports[`test should render correctly 1`] = `
<div
  className="checkbox"
  style={undefined}>
  <label>
    <input
      checked={false}
      disabled={false}
      type="checkbox"
      value={false} />
    Check this
  </label>
</div>
`;
